code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -473,6 +473,11 @@
 /datum/species/vox/armalis/handle_reagents() //Skip the Vox oxygen reagent toxicity. Armalis are above such things.
 	return 1
 
+#define KIDAN_LANTERN_HUNGERCOST 0.5
+#define KIDAN_LANTERN_MINHUNGER 150
+#define KIDAN_LANTERN_LIGHT 4
```

Comment:
This should be moved to be with the organ, then undefined at the end of the file.